os_migrate/plugins/modules/os_conversion_host_info.py

```diff
@@ -116,7 +116,13 @@ def main():
         server = conn.get_server(name_or_id=srv, filters=filters)
         if not server:
             module.fail_json(msg='Conversion host ' + srv + ' not found!')
-        conversion_host['address'] = server.accessIPv4
+        if server.public_v4 == "":
+            module.fail_json(msg='Conversion host address cant be empty \n' + str(server))
+        # The value of the server.accessIPv4 property is not defined by OpenStack,
+        # and may not be consistent across vendors.
+        # TODO:DISCUSSION Should we use server.public_v4 instead?
+        # conversion_host['address'] = server.accessIPv4
+        conversion_host['address'] = server.public_v4
```

Comment:
I'm not familiar with these properties but just want to comment here to mark it as "this warrants a closer look before merging". Specifically i wonder if the "public_v4" name is generic, or if it is just generated because our external network happens to be named "public", in which case this would break the migration for other environments.